### ISSUE TITLE
Enum documentation

### ISSUE THREAD

**Issue by kerimoyle** [libcellml/website-src#9]
I'm not sure whether this is an issue for this repo or the libcellml code one, but it would be good to either make use the enum documentation for values or remove it from display in the built pages.  At present, we don't document enum values in the source code:

![image](https://user-images.githubusercontent.com/46887220/100158797-9d017500-2f11-11eb-85c1-84ab5623285c.png)

... but we do include an empty "description" column in the built documents:

![image](https://user-images.githubusercontent.com/46887220/100158887-c0c4bb00-2f11-11eb-981f-e17c59f3cd9b.png)

Either document all enum values, or remove the second column.

**Comment by hsorby** [comment]
I don't think it is for here.  Either cellml/libcellml or change hsorby/vue-doxygem-xml to only add description column if at least one entry has a description.
